src/pages/contacts/contacts.js

```diff
@@ -62,7 +62,10 @@ const ContactsPage = () => {
             </span>
             <div className={styles.schedule}>
               {contact.openHours[language].value.split('|').map((el, i) => (
-                <span key={i}>{el}</span>
+                <div key={i}>
```

Comment:
Please, don't use for a key index of the array, use some unique element from the element